diagnostic.md

```diff
@@ -6,70 +6,72 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+With curl and from browser
+I were not able to write curl command 
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+The requests first will be going to routes to see if the resources are defined, Then it will be going to controller, to modals and modals will be looking into database.
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It executes the scripts in the rake and migrates the database
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.